sfm_pc/management/commands/import_osm.py

```diff
@@ -348,9 +355,10 @@ def downloadPBFs(self, country):
 
     def downloadBoundaries(self, country):
 
-        file_path = os.path.join(self.data_directory, '{}.tgz'.format(country['country']))
+        file_path = os.path.join(self.data_directory, '{}.zip'.format(country['country']))
+        url = 'https://wambachers-osm.website/boundaries/exportBoundaries?cliVersion=1.0&cliKey=e05676a7-72dc-4367-b8ed-6207c43926b4&exportFormat=json&exportLayout=levels&exportAreas=water&union=false&from_AL=2&to_AL=8&selected='
```

Comment:
You'll want to make the API Key a setting in the settings file. Probably the local settings file so that we can have different keys for different environments. We should also make the base URL for downloading these shapes a setting in case they change that at some point. Once you have that all sorted out, we can use python string formatting to make the correct URL:

something like:
```
"{base_url}?cliVersion=1.0&cliKey={api_key}&exportFormat=json&exportLayout=levelsexportAreas=water&union=false&from_AL=2&to_AL=8&selected={relation_id}".format(
    base_url=settings.OSM_BASE_URL,
    api_key=settings.OSM_API_KEY,
    relation_id=relation_id)
```